zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/DecisionRequirementsQueryController.java

```diff
@@ -84,11 +72,9 @@ public ResponseEntity<DecisionRequirementsItem> getByKey(
           .body(
               SearchQueryResponseMapper.toDecisionRequirements(
                   decisionRequirementsServices.getByKey(decisionRequirementsKey)));
-    } catch (final Exception exc) {
-      REST_LOGGER.warn("An exception occurred in get Form by key.", exc);
-      final var problemDetail =
-          RestErrorMapper.mapErrorToProblem(exc, RestErrorMapper.DEFAULT_REJECTION_MAPPER);
-      return RestErrorMapper.mapProblemToResponse(problemDetail);
+    } catch (final Exception e) {
+      REST_LOGGER.debug("An exception occurred in getDecisionRequirements.", e);
```

Comment:
💭 The stack trace will already tell us where the exception occurred, right? Could we simply add the logging to the error mapping method so it's handled for all callers?

Reply:
Good idea 👍 
I just figured out that most of the exceptions are logged in https://github.com/camunda/camunda/blob/befb8e130097dbe154ebc7458108e73f5cf45d8a/zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/RestErrorMapper.java#L86-L175

I will add some debug/trace logs for the missing ones